about.html

```diff
@@ -0,0 +1,187 @@
+               <h1 class="hoo">CONCERT 2021</h1>
+            </div>
+
+            <div class="intro-container">
+              <div class="about-introduction">
+                The Big Bang concert will be the greatest of all times, you can't afford to miss it, great musicians from all over the world we be live perform. This is will be a beautiful and happy moment for all of us. It will take place in NewYork. THis will be a great hit.
+              </div>
+             </div>
+
+            <div class="contact">
+                <p class="poo">please contact us per Email for any further question</p>
+                <p class="poo">about BB Big Bang 2022!</p>
+                <p class="poo" id="mail">bbBigBang2021@bbnewYork.ent</p>
+            </div>
+
+    </header> 
+
+    <section class="mission">
+        <h2 class="topic">Big Bang</h2>
+
+        <P id="topic">BB Big Bang Concert 2021 Logo</P>
+
+        <h2 id="topic-desk">BB Big Bang COncert 2021 Logo</h2>
+
+        <div class="lining">
+            <div class="line">.</div>
+        </div>
+
+        <p id="about-logo">The logo of BB Big Bang Concert 2021 was decided through the logo competition from 8.June to 7.july. After all the issue, we tried to solve it all but for we where able to figure them out, thank you so much i really appreciate</p>
+
+        <div class="logo">
+            <img id="logo-img" src="Images/the-logo.png" alt="">
+        </div>
+    </section>
+
+    <section class="past">
+        <h2> See the past BB Big Bang Concert</h2>
+        <div class="lining">
+            <div class="line">.</div>
+        </div>
+
+        <p class="last-two">
+            Take a look at the last two weeks BB Big Bang Concert Logo which took place in Buenos Aires and in Warsaw.
+        </p>
+
+        <div class="past-img-container">
+
+        <div class="past-img">
+
+           <img class="past-img1" src="Images/2020.jpg" alt="">
+
+           <div class="year-place">
+
+           <p class="year">2020</p>
+           <p  class="place">BB Big Bang Concert 2020 in Buenas Aires</p>
+          </div>
+        </div>
+
+        <div class="past-img">
+
+            <img class="past-img1" src="Images/2019.jpg" alt="">
+ 
+            <div class="year-place">
+ 
+            <p class="year">2019</p>
+            <p class="place">BB Big Bang Concert 2019 in Buenas Aires</p>
+           </div>
+         </div>
+
+        </div>
+
+         </section>
+
+    <section class="partners">
+
+        <h1>Partners</h1>
+
+         <div class="lining">
+            <div class="line">.</div>
+        </div>
+ 
+        <div class="row1">
+        <p class="partner1">Topie</p>
+        <p class="partner2">Kindom</p>
+        <p class="partner3">NEBER</p>
+        </div>
+
+        <div class="row2">
+            <p class="partner4">damamkao</p>
+            <p class="partner5">Mlalae</p>
+        </div>
+        
+    </section>
+    
+    <div class="logo-reservedright">
```

Comment:
Kindly make sure that your footer has a dark background for your about page 🔥 This way we follow the project requirements 🙌

Reply:
DONE